dist/img/priority_icon.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="14.432" height="13.864" viewBox="0 0 14.432 13.864"><defs><style>.a{fill:none;stroke:#a3a3a3;stroke-linecap:round;stroke-linejoin:round;stroke-width:1.2px;}</style></defs><g transform="translate(0.832 0.832)"><line class="a" x1="5.721" transform="translate(5.843 6.128)"/><line class="a" x1="7" transform="translate(6 3)"/><line class="a" x1="4.196" transform="translate(6.15 9.28)"/><line class="a" x1="2.67" transform="translate(6.246 12.432)"/><line class="a" y1="12" transform="translate(2)"/><line class="a" x1="2" y1="3" transform="translate(2)"/><line class="a" y1="3" x2="2"/></g></svg>
```

Comment:
if you don't use this file, just remove it?

Reply:
finished